backend/fms_core/services/project.py

```diff
@@ -39,4 +43,38 @@ def create_project(name=None, principal_investigator=None, requestor_name=None,
     except ValidationError as e:
         errors.append(str(e))
 
-    return (project, errors, warnings)
\ No newline at end of file
+    return (project, errors, warnings)
+
+def add_sample_to_study(sample: Sample, project: Project, study_letter: str, step_order: int | None = None):
```

Comment:
Include a comment for the automated documentation and add tests for the service.

Reply:
Automated documentation... I totally forgot we had that, even though I worked on that...